Modules/TRD/calibParamTRD.json

```diff
@@ -0,0 +1,82 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080"
+      },
+      "Activity": {
+        "number": "526486"
+      },
+      "conditionDB": {
+        "url": "alice-ccdb.cern.ch"
+      },
+      "postprocessing": {
+        "matchAnyRunNumber": "true"
+      }
+    },
+    "postprocessing": {
+      "CalibTrendTRD": {
+        "active": "true",
+        "className": "o2::quality_control::postprocessing::TrendingTask",
+        "moduleName": "QualityControl",
+        "detectorName": "TRD",
+        "producePlotsOnUpdate": "false",
+        "dataSources": [
+          {
+            "type": "condition",
+            "path": "TRD/Calib",
+            "names": [
+              "CalVdriftExB"
+            ],
+            "reductorName": "o2::quality_control_modules::trd::CalibReductorTRD",
+            "moduleName": "QcTRD"
+          }
+        ],
+        "plots": [
+          {
+            "name": "vdrift_trend",
+            "title": "vDrift mean trend",
+            "varexp": "CalVdriftExB.vdriftmean:time",
+            "selection": "",
+            "option": "*L",
+            "graphYRange": "0:3.5",
+            "graphXRange": "",
+            "graphAxisLabel": "vdrift:time",
+            "graphErrors": "0:CalVdriftExB.vdrifterr"
+          },
+          {
+            "name": "vdrift_trend2",
+            "title": "vDrift trend",
+            "varexp": "CalVdriftExB.vdrift[]:runNumber",
+            "selection": "",
+            "option": "colz logz",
+            "graphYRange": "-2:3.5",
+            "graphXRange": "",
+            "graphAxisLabel": "vdrift:time"
```

Comment:
But this still looks like the definition of a graph. Can't you also set the number of bins for example?

Reply:
it doesn't seem to have any option, related to binning. i am working on implementing our own trending task 🙂